profiles/dini.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+const modules = [
+  // order is significant
+  import("../src/core/base-runner.js"),
+  import("../src/core/ui.js"),
+  import("../src/core/location-hash.js"),
+  import("../src/core/l10n.js"),
+  import("../src/dini/defaults.js"),
+  import("../src/core/style.js"),
+  import("../src/w3c/style.js"),
+  import("../src/core/data-include.js"),
+  import("../src/core/markdown.js"),
+  import("../src/core/reindent.js"),
+  import("../src/dini/headers.js"),
+  import("../src/w3c/abstract.js"),
```

Comment:
I'd suggest copy/pasting this into a dini/abstract.js ... there is risk that we might break the DINI profile if we make changes to this file.  

```suggestion
  import("../src/w3c/abstract.js"),
```

Reply:
Done with https://github.com/w3c/respec/pull/2739/commits/2aa34468932b8614e96ebc53a4760ab3bba9f8a9